src/cpu/cpu_softmax_list.cpp

```diff
@@ -46,6 +46,7 @@ const std::map<pk_impl_key_t, std::vector<impl_list_item_t>> &impl_list_map() {
             CPU_INSTANCE_X64(jit_uni_softmax_fwd_t)
             CPU_INSTANCE_AARCH64(jit_uni_softmax_fwd_t<sve_512>)
             CPU_INSTANCE_AARCH64(jit_uni_softmax_fwd_t<sve_256>)
+            CPU_INSTANCE_AARCH64(jit_uni_softmax_fwd_t<sve_128>)
```

Comment:
Minor: please consider removing templated argument for softmax implementation as with time it becomes bulky.

Reply:
Thank you for the comment. I'd like to improve templated argument in future PR.